egs/madcat_ar/v1/local/create_line_image_from_page_image.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/env python3
+# Minimum Bounding Box part in this script is originally (https://github.com/BebeSparkelSparkel/MinimumBoundingBox)
+
+
+import argparse
+import os
+import xml.dom.minidom as minidom
+from PIL import Image
```

Comment:
We have to decide if this (PIL) is the image library we want to use and maybe we should have a script that checks if it and other dependencies are installed (called from the run.sh) and an installation script?  It might be just a case of doing "pip install Pillow".  (?).  I notice PIL is not available by default on my mac-- it must not be a core Python library.  @hhadian, comments?  For now this doesn't have to be changed though.

Also, scipy is a big dependency and probably one whose presence should be checked by the script.  It might turn out to be easier just to write our own convex-hull code.  But for now it's not a priority to address this.

Reply:
I guess Pillow (which is the new PIL) is not very actively maintained. **scikit-image** is more up-to-date but also bigger I guess. I'll need to do some checking to see what other options we might have.
[P.S. I just tried installing Pillow on my fedora and it failed but I could easily install scikit-image.]

For checking, Ashish could add a similar script to `egs/iam/v1/local/check_tools.sh` to check for all the requirements.